pyproject.toml

```diff
@@ -22,10 +22,11 @@ dependencies = [
     'photutils >=1.6.0',
     'pyparsing >=2.4.7',
     'requests >=2.22',
-    # 'rad >=0.15.0',
-    'rad @ git+https://github.com/spacetelescope/rad.git@main',
-    # 'roman_datamodels >=0.15.0',
-    'roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git@main',
+    'rad == 0.17.0',
```

Comment:
I think this pin is causing the devdeps job on plwishmaster to pull the older, 0.17.0 rad (and similar for roman_datamodels) instead of testing against the 0.17.1 (or development version, which I think is the same at the moment).

@ddavis-stsci was this a requirement for this PR? If so, is there a follow-up PR or tracking issue to remove the pin?